challenge.md

```diff
@@ -31,21 +31,15 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+This is an absolute path because it contains the root followed by the user. A relative path would look something like this: /directoryexample/subdirectoryexample
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+An absolute path. Relative paths work best within the computer, but files outside the computer (EX: On the internet) should use absolute paths.
```

Comment:
Correct! Actually, we don't have a choice about using a relative path here - things on the internet are referenced by their URLs, and a URL is an example of an absolute path.